Source/Extensions/Blazorise.Components/DropdownList.razor.cs

```diff
@@ -168,6 +212,11 @@ protected bool IsSelected( TValue value )
     /// </summary>
     [Parameter] public EventCallback<TValue> SelectedValueChanged { get; set; }
 
+    /// <summary>
+    /// Enebles filter text input on the top of the items list.
+    /// </summary>
+    [Parameter] public bool FilterEnabled { get; set; }
```

Comment:
We don't usually have `FeatureEnabled`, with `*Enabled` sufix naming style. So I would propose:

- `Filterable` (similar to what we have on DataGrid)
- `ShowFilter` (similar `Show*` prefix to what we have on other components)
- `Searchable`, or `ShowSearch` as an alternative for filter.

Reply:
Filterable it is.